src/site/stages/build/process-cms-exports/transformers/helpers.js

```diff
@@ -122,4 +113,35 @@ module.exports = {
   utcToEpochTime(timeString) {
     return moment.tz(timeString, 'UTC').unix();
   },
+
+  createMetaTagArray(metaTags, typeName = '__typename') {
+    function createMetaTag(type, key, value) {
+      return {
+        [typeName]: type,
+        key,
+        value,
+      };
+    }
+
+    return [
+      createMetaTag('MetaValue', 'title', metaTags.title),
+      createMetaTag('MetaValue', 'twitter:card', metaTags.twitter_cards_type),
+      createMetaTag('MetaProperty', 'og:site_name', metaTags.og_site_name),
+      createMetaTag(
+        'MetaValue',
+        'twitter:description',
+        metaTags.twitter_cards_description,
+      ),
+      createMetaTag('MetaValue', 'description', metaTags.description),
+      createMetaTag('MetaValue', 'twitter:title', metaTags.twitter_cards_title),
+      createMetaTag('MetaValue', 'twitter:site', metaTags.twitter_cards_site),
+      createMetaTag('MetaProperty', 'og:title', metaTags.og_title),
+      createMetaTag('MetaProperty', 'og:description', metaTags.og_description),
+      createMetaTag(
+        'MetaProperty',
+        'og:image:height',
+        metaTags.og_image_height,
+      ),
+    ].filter(t => t.value);
```

Comment:
My big concern was having metatags for missing data, so I'm glad that this is here.